### ISSUE TITLE
EMS function @RhovFnTdbRhLBnd0C is being parsed as @RhovFnTdbRh

### ISSUE THREAD

**Issue by EnergyArchmage** [NREL/EnergyPlus#5284]
While working on another issue (to detect unitialized variables), found that RhovFnTdbRhLBnd0C is actually being parsed as RhovFnTdbRh and a variable LBnd0C.   The substrings match and parser gets confused.  

function RhovFnTdbRhLBnd0C will not work in EMS programs.

**Comment by rraustad** [comment]
Just move the latter up above so it's checked first ?

```
} else if ( SameString( String.substr( Pos, 12 ), "@RhovFnTdbRh" ) ) {
    if ( DeveloperFlag ) gio::write( OutputFileDebug, fmtA ) << "FUNCTION \"" + String.substr( Pos, 12 ) + "\"";
    Token( NumTokens ).Operator = FuncRhovFnTdbRh;
    Token( NumTokens ).String = String.substr( Pos, 12 );
    Pos += 11;
} else if ( SameString( String.substr( Pos, 18 ), "@RhovFnTdbRhLBnd0C" ) ) {
    if ( DeveloperFlag ) gio::write( OutputFileDebug, fmtA ) << "FUNCTION \"" + String.substr( Pos, 18 ) + "\"";
    Token( NumTokens ).Operator = FuncRhovFnTdbRhLBnd0C;
    Token( NumTokens ).String = String.substr( Pos, 18 );
    Pos += 17;
```

**Comment by rraustad** [comment]
I assume there is a similar issue here ?

```
} else if ( SameString( String.substr( Pos, 12 ), "@RhFnTdbRhov" ) ) {
    if ( DeveloperFlag ) gio::write( OutputFileDebug, fmtA ) << "FUNCTION \"" + String.substr( Pos, 12 ) + "\"";
    Token( NumTokens ).Operator = FuncRhFnTdbRhov;
    Token( NumTokens ).String = String.substr( Pos, 12 );
    Pos += 11;
} else if ( SameString( String.substr( Pos, 18 ), "@RhFnTdbRhovLBnd0C" ) ) {
    if ( DeveloperFlag ) gio::write( OutputFileDebug, fmtA ) << "FUNCTION \"" + String.substr( Pos, 18 ) + "\"";
    Token( NumTokens ).Operator = FuncRhFnTdbRhovLBnd0C;
    Token( NumTokens ).String = String.substr( Pos, 18 );
    Pos += 17;
```

**Comment by EnergyArchmage** [comment]
Probably, yes. good catch.

**Comment by rraustad** [comment]
See PR #5763

**Comment by EnergyArchmage** [comment]
closed via PR #5763
